packages/performance/src/services/settings_service.ts

```diff
@@ -88,4 +102,22 @@ export class SettingsService {
     }
     return settingsServiceInstance;
   }
+
+  // True if event should be sent to transport endpoint rather than log endpoint.
+  // rolloutPercent is in range [0.0, 100.0].
+  static isDestTransport(iid: string, rolloutPercent: number): boolean {
```

Comment:
These two method do not  need to be on settings_service class and can be pure function in whichever class that uses them.